ops/terraform/env/mgmt/terraform.tf

```
@@ -39,7 +39,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.67"
+      version = "~> 5.53.0" #"~> 4.67"
```

I'm OK with updating providers as part of these changes since `mgmt` is typically applied out-of-band manually. I think we can remove the commented out version here, though.

```suggestion
      version = "~> 5.53.0"
```